example/generator/generator_run.sh

```diff
@@ -16,7 +16,13 @@ fi
 
 setup_interfaces
 
-./odp_generator${EXEEXT} -w 1 -n 1 -I $IF0 -m u
+if [ "$(which stdbuf)" != "" ]; then
+	STDBUF="stdbuf -o 0"
+else
+	STDBUF=
+fi
+
+$STDBUF ./odp_generator${EXEEXT} -w 1 -n 1 -I $IF0 -m u
```

Comment:
When we have multiple sub applications running as a part of test(triggered by pktio_env), we need STDBUF separately for each application since it would be redirected to separate files

Reply:
But stdbuf is not specific to any file, it modifies stream buffering for a process. And the buffering is inherited, so it applies to child processes as well. So it could be on a higher level, if it's needed at all.

I'm still not sure what problem you're trying to fix. Seems like the effect of modifying the buffering would basically be that some prints are written to files a few seconds earlier.